autoware.repos

```diff
@@ -0,0 +1,53 @@
+repositories:
+  autoware/autoware:
+    type: git
+    url: git@github.com:autowarefoundation/autoware.git
+    version: master
+  autoware/core_control:
+    type: git
+    url: git@github.com:autowarefoundation/core_control.git
+    version: master
+  autoware/core_perception:
+    type: git
+    url: git@github.com:autowarefoundation/core_perception.git
+    version: master
+  autoware/core_planning:
+    type: git
+    url: git@github.com:autowarefoundation/core_planning.git
+    version: master
+  autoware/documentation:
+    type: git
+    url: git@github.com:autowarefoundation/documentation.git
+    version: master
+  autoware/messages:
+    type: git
+    url: git@github.com:autowarefoundation/messages.git
+    version: master
+  autoware/simulation:
+    type: git
+    url: git@github.com:autowarefoundation/simulation.git
+    version: master
+  autoware/utilities:
+    type: git
+    url: git@github.com:autowarefoundation/utilities.git
+    version: master
+  autoware/visualisation:
+    type: git
+    url: git@github.com:autowarefoundation/visualisation.git
```

Comment:
We'll have to come up with a style guide for spelling as well. I don't mind we either adopt the Commonwealth/British or American spellings (I'm not a native speaker, so I don't have much of a say), as long as we are mostly consistent.

Reply:
Alas, I didn't even think about it when I used the *correct* English spelling. :wink:

The Japanese use <strike>Broken</strike>American English, and we have many users in the USA, so maybe that's the way to go. I don't want to waste time choosing the language for the bike shed signs, so I'm just going to go with that. It will give us consistency with ROS, too.

Reply:
That's fine. The target directory still points to `autoware/visualisation`, though.